05_syntax_lab/02.py

```diff
@@ -4,3 +4,31 @@
 If the sum is divisible by 7, also print the word "Boom".
 """
 
+import random
+sum = 0
+
+for number in range(7):
+    print random.randint(1,100)
+    sum += number
+
+print "sum =", sum
```

Comment:
Sum calculation does not calculate total of all the random numbers, but always prints 21. Can you figure out why?